package/src/test.rs

```diff
@@ -334,6 +372,31 @@ fn test_pants_bootstrap_tools(scie_pants_scie: &Path) {
     .unwrap();
 }
 
+fn test_pants_2_25_using_python_3_11(scie_pants_scie: &Path) {
+    integration_test!("Verifying we can run Pants 2.25+, which uses Python 3.11");
+    if is_macos_thats_too_old(13, 14) {
+        log!(
+            Color::Yellow,
+            "Pants 2.25 cannot run on this version of macOS => skipping"
+        );
```

Comment:
It would be nice, if not in the message, but at least as a code comment, the reason we can't run pants 2.25 on these older versions of MacOS.